### PR TITLE
ci: notify after build step on new merges to master

### PR DESCRIPTION
**Pull request by psantos-consensys**

### CODE REVIEW

File: .github/workflows/node.js.yaml [hunk 1]
```diff
@@ -1,57 +1,85 @@
...
+        with:
+          name: codecov-umbrella
+          directory: ./coverage/
+          fail_ci_if_error: true
+          verbose: true
+
+  notify-slack:
+    needs: build
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```
Comment (blitz-1306) [inline .github/workflows/node.js.yaml:62]: Can we also extend it to inform about merges to `develop`? We use two branches in this repository.

**Review by cd1m0 — CHANGES_REQUESTED** [review]
@psantos-consensys could you just address Pavel's comment below? Then it will be ready to merge

File: .github/workflows/node.js.yaml (hunk 1 above)
Comment (psantos-consensys) [reply .github/workflows/node.js.yaml]: removed the if statement and modified the env to pick the branch name

**Review by cd1m0 — DISMISSED** [review]
lgtm

**Review by blitz-1306 — APPROVED** [review]

**Review by blitz-1306 — APPROVED** [review]

**Pull request merged**